apps/web/composables/useCarousel/useCarousel.ts

```diff
@@ -0,0 +1,61 @@
+import { onMounted, onBeforeUnmount } from 'vue';
+
+const handleArrowsVisibility = () => {
+  const nextButton = document.querySelector('.swiper-button-next') as HTMLElement;
+  const previousButton = document.querySelector('.swiper-button-prev') as HTMLElement;
+
+  if (window.innerWidth < 768) {
```

Comment:
**issue**: We should handle breakpoints via `useViewport()`.

Reply:
1. Why is this a issue? 2. Why do we need to overcomplicated with useViewport() in this straightforward case?

Reply:
I think we define breakpoints via `useViewport` everywhere, so for consistency. Also, if the viewpoint configuration ever gets updated, nobody would think to check for a hard coded value in this composable.